proto/umee/incentive/v1/genesis.proto

```diff
@@ -21,59 +21,55 @@ message GenesisState {
   repeated AccountUnbondings account_unbondings  = 10 [(gogoproto.nullable) = false];
 }
 
-// TotalBond tracks the amount of coins of one uToken denomination bonded to a
-// given reward tier by all accounts. Used by queries TotalBonded and TotalUnbonding.
-message TotalBond {
-  uint32                   tier   = 1;
-  cosmos.base.v1beta1.Coin amount = 2 [(gogoproto.nullable) = false];
-}
-
-// Bond tracks the amount of coins of one uToken denomination bonded to a
-// given reward tier by a single account.
+// Bond tracks the amount of coins of one uToken denomination bonded
+// by a single account.
 message Bond {
   string                   account = 1;
-  uint32                   tier    = 2;
-  cosmos.base.v1beta1.Coin amount  = 3 [(gogoproto.nullable) = false];
+  cosmos.base.v1beta1.Coin uToken  = 2 [(gogoproto.nullable) = false];
 }
 
-// RewardTracker tracks the value of a given tier and lock denom's RewardAccumulator
-// at the last time a specific account calculated pending rewards for it. When calculating
+// RewardTracker tracks the value of a given lock denom's RewardAccumulator at the
+// ast time a specific account calculated pending rewards for it. When calculating
 // available rewards, this value is used to determine the difference between the current
-// RewardAccumulator for a tier and the last value at which the user updated bonds or claimed
+// RewardAccumulator for a uToken and the last value at which the user updated bonds or claimed
 // tokens. Their pending rewards increase by only the rewards accrued in that time period.
 message RewardTracker {
-  string   account                                    = 1;
-  uint32   tier                                       = 2;
-  string   denom                                      = 3;
-  repeated cosmos.base.v1beta1.DecCoin reward_tracker = 4 [
+  string   account                             = 1;
+  string   denom                               = 2;
+  repeated cosmos.base.v1beta1.DecCoin rewards = 3 [
```

Comment:
They are ratios which multiply the user's actual bonded amount, so decimals are useful (and necessary when rewards per block are less than 1 base token but nonzero, which might be common).

Commenting the relevant info from the spec:

```go
ra := RewardAccumulator{
   denom: "u/uumee",
   exponent: 6,
   rewards: "0.00023uumee, 0.00014ibc/1234",
}
```

The example reward accumulator above can be interpreted as:

> If `10^6` `u/uumee` was bonded at genesis and had remained bonded since, it would have accumulated `0.00023uumee` and `0.00014ibc/1234` in rewards.

Reply:
It shouldn't depend how much was bonded , right? The total accumulator will always have same amount of new rewards in each period of time. So it should only depend on time.

Reply:
The increase per block depends on bonded tokens.

For example, assuming one active incentive program which awards `1uumee` per block spread across bonded `uatom` suppliers.

- If 10^6 `uatom` is bonded on a given block, then `RewardAccumulator(`uatom`) increases by `1uumee`
- If 10^7 `uatom` is bonded on a given block, then `RewardAccumulator(`uatom`) increases by `0.1uumee` since the same reward is being spread over a larger supply

Reply:
Oh, I see. So it's not the accumulator of total rewards, but accumulator * shares. Can we update a comment?